traitsui/tests/editors/test_tree_editor.py

```diff
@@ -110,6 +153,34 @@ def _test_tree_editor_releases_listeners(self, hide_root, nodes=None,
             notifiers_list = bogus.trait(trait)._notifiers(False)
             self.assertEqual(0, len(notifiers_list))
 
+    def _test_tree_node_object_releases_listeners(self, hide_root, nodes=None,
+                                                  trait='bogus_list',
+                                                  expected_listeners=1):
```

Comment:
None of the calling code uses the `expected_listeners` arg. You should probably remove it.